frontend/src/components/UploadArea/UploadArea.tsx

```diff
@@ -49,6 +52,17 @@ const UploadArea: FunctionComponent<UploadingAreaProps> = ({ value, onChange }):
             uploadInputRef.current?.click();
           }
         }}
+        onDragOver={async e => {
+          e.preventDefault();
+        }}
+        onDrop={async e => {
+          e.preventDefault();
+          e.stopPropagation();
+          if (e.dataTransfer.files && e.dataTransfer.files.length > 0) {
```

Comment:
Does it permit uploading more than 1 file? I have no idea what I am writing, but shouldn't we check that `length == 1`?